src/dlgprefsound.cpp

```diff
@@ -136,6 +136,14 @@ DlgPrefSound::DlgPrefSound(QWidget* pParent, SoundManager* pSoundManager,
             this, SLOT(masterOutputModeComboBoxChanged(int)));
     m_pMasterMonoMixdown->connectValueChanged(this, SLOT(masterMonoMixdownChanged(double)));
 
+    m_pMasterTalkoverMix = new ControlObjectSlave("[Master]", "talkover_mix", this);
+    micMixComboBox->addItem(tr("Enabled"));
```

Comment:
```Microphone Mix: Enabled``` seems a little confusing. I wonder what a new user would think that meant? 

It also doesn't indicate that if another channel becomes a talkover channel it will bypass master too. 

Throwing out some ideas:
```Microphone/Talkover Output: Master output, Broadcast, and Recording```
```Microphone/Talkover Output: Broadcast and Recording Only```

```Microphone/Talkover Mixing: Master output```
```Microphone/Talkover Mixing: Broadcast and Recording Only```

@daschuer @esbrandt @ywwg @kain88-de thoughts?